README.md

```diff
@@ -1,2 +1,276 @@
+multiple hardware architectures, optimized for resource constrained devices,
+and built with security in mind.
+
+The Zephyr build system supports integration of externally maintained
+projects called *modules*. This integration is configured through the
+settings in a *module.yml* file in a :file:`zephyr/` folder at the root
+of the repository.  See the [Zephyr Modules (External projects)][5]
+for a high-level overview of the build system module mechanism.
+
+
+## Scope
+
+The Zephyr eXample Application (ZXA) module is supplementary to the Zephyr
+repository.  This provides:
+
+1. A reference implementation of using the Zephyr module mechanism.
+
+1. A repository for holding example code and config files outside of the
+   Zephyr repository, including:
+
+   - `west` manifest (in the default location)
+   - projects/samples/tests
+   - applications/subsystems/drivers/interface declarations
+   - devicetree source files
+   - boards
+   - scripts
+
+1. A test case for CI to monitor Zephyr build system support for modules.
+
+
+   _WARNING:_ The Zephyr repository is intentionally
+   **independent of and unaware of this repository**.
+
+
+# Getting Started
+
+## Creating a New Module
+
+This module is designed to be a boilerplate for other modules simply by:
+
+1. cloning
+
+1. stripping out any unnecessary content (e.g. no drivers or subsystems)
+
+  - If you don't have any non-sample, non-test projects:
+
+    1. remove the `zephyr/projects/` subdirectory
+    1. In `zephyr/modules.yml` remove the reference to `zephyr/projects`
+
+
+## Creating a Free-Standing Manifest Repository
+
+This example module is also an appropriate template for creating a
+manifest-only repository because, like modules that also contain a
+manifest file, they may contain the following:
+
+- documentation describing the scope and purpose of the workspace
+  configured by the manifest
+
+- configuration settings for CI pipelines
+
+- scripts unique to the breadth of the workspace
+
+To strip this repository down to be a manifest-only repository:
+
+  1. Remove all files and subdirectories referenced by `zephyr/module.yml`
+  1. Remove the `zephyr/include` subdirectory
+  1. Remove `zephyr/module.yml`
+
+
+## Checking Out Code
+
+The Zephyr module ZXA may be cloned and added to the Zephyr build system
+in exactly the same way as any other repository.
+
+Once the Zephyr development environment has been set up (see [Zephyr Getting Started Guide][3]), then follow instructions here to:
+
+- Get the source code with Zephyr module ZXA as the `west`
+  manifest repository
+- Build and run projects
+
+
+## Get Zephyr and Install Python Dependencies
+
+Clone Zephyr module ZXA and its dependencies into a new `west`
+workspace named :file:`zp`. You'll also install additional Python
+dependencies.
+
+1. Get the Zephyr source code:
+
+      ```bash
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        ~/zxa
```

Comment:
`~/` doesn't work on Windows. Please see the Zephyr GSG and follow the example there instead.

Reply:
I started out with documentation in RST format and tabs for each of the development platforms.  The feedback I received in this PR was to use MD instead and to describe it for only one platform to simplify the documentation.  This sounds like contradictory requests. 

I have not found any MD files in Zephyr that provide command examples for each of the host environments.  (The closest is `scripts/tests/twister/README.md`, which only provides `bash` examples.) 

Please either provide an example of how you want this formatted in MD, or allow it to be in RST.